qiskit/synthesis/__init__.py

```diff
@@ -89,6 +89,28 @@
 
 .. autofunction:: synth_qft_line
 
+Unitary Synthesis
+=================
+
+.. autofunction:: qs_decomposition
+
+One-qubit Synthesis
```

Comment:
Tiny nitpick, but the capitalization could be more consistent, e.g., either `One-Qubit Synthesis` or `One-qubit synthesis` 🙂